Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.h

```diff
@@ -390,7 +390,9 @@ namespace AZ
             }
 
             const PipelineLayout& pipelineLayout = pipelineState->GetPipelineLayout();
+#if defined (AZ_RHI_ENABLE_VALIDATION)
             const RHI::PipelineLayoutDescriptor& pipelineLayoutDescriptor = pipelineLayout.GetPipelineLayoutDescriptor();
```

Comment:
why not move that down under the same block (line 504)?  it's not used anywhere else